### PR TITLE
Add organization fields on lessor model

### PR DESCRIPTION
**Pull request by neungkl**

### CODE REVIEW

File: webserver/apps/lessors/serializers.py [hunk 1]
```diff
@@ -1,21 +1,57 @@
...
         instance.save()
         return instance
+
+    def validate_organization_phone_number(self, data):
+        if (data is None) or re.match(r"^\+?\d{9,15}$", data):
+            return data
+        raise serializers.ValidationError('Phone number should be in format +123456789')
+
+    def validate(self, data):
+        print(data)
```
Comment (futurizing) [inline webserver/apps/lessors/serializers.py]: Remove

File: webserver/apps/lessors/models.py [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
-    lessor_name = models.CharField(verbose_name='Lessor Name', max_length=100)
+    lessor_name = models.CharField(verbose_name="Lessor Name", max_length=100)
+
+    is_organization = models.BooleanField(verbose_name="Is Organization", default=False)
+
+    organization_name = models.CharField(verbose_name="Organization Name", max_length=100, null=True)
+    organization_contact_name = models.CharField(verbose_name="Organization Contact Name", max_length=100, null=True)
+    organization_email = models.EmailField(verbose_name="Organization Email", max_length=254, null=True)
+    organization_phone_number = models.CharField(verbose_name="Organization Phone Number", max_length=15, null=True)
+    # organization_phone_number = models.RegexField(regex=r"^\+?\d{9,15}$", verbose_name="Organization Phone Number", min_length=9, max_length=15)
```
Comment (futurizing) [inline webserver/apps/lessors/models.py]: Remove

File: webserver/apps/lessors/models.py [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
 from apps.users.models import User
 
-
 class Lessor(models.Model):
-    lessor_name = models.CharField(verbose_name='Lessor Name', max_length=100)
+    lessor_name = models.CharField(verbose_name="Lessor Name", max_length=100)
+
+    is_organization = models.BooleanField(verbose_name="Is Organization", default=False)
+
+    organization_name = models.CharField(verbose_name="Organization Name", max_length=100, null=True)
```
Comment (sirinthra-cc) [inline webserver/apps/lessors/models.py]: CharField: should set blank=True instead of null=True

Comment (neungkl) [reply webserver/apps/lessors/models.py]: Fixed

File: webserver/apps/lessors/serializers.py (hunk 1 above)
Comment (neungkl) [reply webserver/apps/lessors/serializers.py]: Fixed

**Comment by neungkl** [comment]
Please review and check again @sirinthra-cc @futurizing

**Comment by neungkl** [comment]
Approve this PR, please @sirinthra-cc

**Pull request merged**